src/client/feed/feedReducer.js

```diff
@@ -0,0 +1,12 @@
+import { UPDATE_FEED_RESULTS } from '../search/SearchActions';
+
+const feed = (state = { 'posts': [] }, action) => {
+  switch (action.type) {
+    case UPDATE_FEED_RESULTS:

```

Comment:
So Agree with the constant...

I think it's the searcher's responsibility to make Requests ... In the case you describe, I would say that the Feed in the Results section would have an action that trigger's a search request to refresh results or something along those lines.

I just think the feed stuff should be like FILTER_WORD  or FILTER_USER rather than retrieving and requesting the search itself. 

I'm also wary of having too many reducer files with 1 or two cases when they could be combined because they make enough sense... then we can break it up if it gets unwieldy later so we don't have to keep jumping between files.